requirements.txt

```diff
@@ -3,5 +3,5 @@
 # Format:
 #
 #   name {repo} {branch|tag}
-libhtp https://github.com/OISF/libhtp 0.5.46
-suricata-update https://github.com/OISF/suricata-update 1.3.0
+libhtp https://github.com/OISF/libhtp 0.5.x
+suricata-update https://github.com/OISF/suricata-update master
```

Comment:
why set this to master? It was 1.3.0 before the release

Reply:
I think this is what Jason and I agreed on. But, I'll let him answer in case I misunderstood.
- The thought is that just like the latest version of `libhtp` goes w a dev branch of `suricata`, latest version of a respective `suricata-update` branch should go as well.
- Given that we don't have a `1.3.x` branch, current `master` of `suricata-update` represents the latest state that we use and bundle with suricata's `7.0.x`.
- Similarly, for suricata `6.0.x`, we use the branch `1.2.x` of `suricata-update` as that reflects the latest state of what should be used and bundled.
- Please note that for now, in `suricata-update`,
branch `master` == tag `1.3.0` and,
branch `master-1.2.x` == tag `1.2.8`

Reply:
Ok, thanks for clearing that up.

Reply:
yes, we already do that :)
**Prep stage**: the requirements.txt gets an update of specific `libhtp` and `suricata-update` version that needs to be bundled in the release. e.g. https://github.com/OISF/suricata/commit/be68bbc4ab288ec84d2fe5e0737ace17ab74ef26#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552
**Post release stage**: the requirements.txt go back to the libhtp and suricata-update branches corresponding to the respective suricata version. e.g. this PR